Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -370,8 +387,9 @@ public static bool PrintHelp(string[] args, Network mainNet)
 				builder.AppendLine("-whitebind=<ip:port>	Bind to given address and whitelist peers connecting to it. Use [host]:port notation for IPv6. Can be specified multiple times.");
 				builder.AppendLine("-externalip=<ip>		Specify your own public address.");
 
-				Logs.Configuration.LogInformation(builder.ToString());				
-				return true;
+			    defaults.Log.LoggerFactory.CreateLogger<NodeSettings>().LogInformation(builder.ToString());
```

Comment:
I'd rather this method returned a string to be shown by the caller.
Logging to show something to the console seems like the wrong thing to do here.